README.md

```diff
@@ -56,14 +57,14 @@ The `setup.py` script automates the setup process, including downloading the rep
 ## Jupyter Notebook Setup (Handled Separately)
 1. How to run the jupyter notebook in virtual environment.
    
-   On MacOS
-   ```bash
-   python3 run_jupyter.py
-   ```
-   Windows and Linux
-   ```bash
-   python run_jupyter.py
-   ```
+   * Navigate to the folder:
+     ```bash
+     cd csusb_fall2024_cse6550_team3/jupyter
+     ```
+   * Run jupyter notebook:
+     ```bash
+     jupyter notebook --port=6003
+     ```
 
```

Comment:
Add instruction to wait for the browser to open. If it doesn't open use this link: http://localhost:6003/tree
Test it out on your end if it works.